administration/src/bp-modules/cards/ImportCardsTable.tsx

```diff
@@ -0,0 +1,56 @@
+import { HotkeysProvider } from '@blueprintjs/core'
+import '@blueprintjs/popover2/lib/css/blueprint-popover2.css'
+import { Cell, Column, Table2, TruncatedFormat2 } from '@blueprintjs/table'
+import '@blueprintjs/table/lib/css/table.css'
+import { useCallback } from 'react'
+import styled from 'styled-components'
+
+import CSVCard from '../../cards/CSVCard'
+import { CardBlueprint } from '../../cards/CardBlueprint'
+import StandaloneCenter from '../StandaloneCenter'
+
+type CardImportTableProps = {
+  headers: string[]
+  cardBlueprints: CardBlueprint[]
+}
+
+const TableContainer = styled(StandaloneCenter)`
+  overflow: auto;
+  flex-basis: 0;
+  padding: 4px 5px;
+  height: 100%;
+`
+
+const CardImportTable = ({ headers, cardBlueprints }: CardImportTableProps) => {
+  const cellRenderer = useCallback(
+    (rowIndex: number, columnIndex: number) => {
+      const cardBlueprint = cardBlueprints[rowIndex] as CSVCard
+      const header = headers[columnIndex]
+      const valid = cardBlueprint.isValueValid(header)
+      const value = cardBlueprint.getValue(header)
+      return (
+        <Cell
+          key={rowIndex + '-' + columnIndex}
+          tooltip={!valid ? 'Validierungsfehler' : undefined}
```

Comment:
It could be possible to show a more sophisticated error message here, but this would only introduce complexity here, so I didn't bother implementing it..

Reply:
i'm fine to keep it simple here